xml/ha_storage_protection.xml

```diff
@@ -782,14 +842,49 @@ Timeout (msgwait)  : 10
     node needs to be fenced. </para>
   </sect1>
 
-<!--  <sect1 xml:id="sec.ha.storage.protect.fencing">
-  <title>Storage-based Fencing</title>
-  <para>
-   You can reliably avoid split brain scenarios by using one or more
-   &stonith; Block Devices (SBD), <literal>watchdog</literal> support and
-   the <literal>external/sbd</literal> &stonith; agent.
-  </para>
- </sect1>-->
+  <sect1 xml:id="sec.ha.storage.protect.test">
+   <title>Testing SBD and Fencing</title>
+   <para>To test if the combination of SBD and fencing works, use one or all
+    of the following methods to check them:
+   </para>
+   <itemizedlist>
+    <listitem>
+     <para>Manually trigger fencing for node <replaceable>NODENAME</replaceable>:</para>
+     <screen>&prompt.root;<command>crm</command> node fence <replaceable>NODENAME</replaceable></screen>
+    </listitem>
```

Comment:
What should be the result of this test? Would be good to mention it, too.

Reply:
Agreed. For example, the fencing target should be fenced and the online nodes consider the fencing target has been fenced after the time stonith-watchdog-timeout.